Src/BootCamp.Chapter/LinqDemo.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace BootCamp.Chapter
+{
+    internal class LinqDemo
+    {
+        internal static void Demo()
+        {
+            var list1 = new List<int> { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };
+            var list2 = new List<int> { 1, 11, 12 };
+            var list3 = new List<int> { 4, 2, 5, 4 };
+
+            // Any. so Im going to check if any number in list1 is even.
```

Comment:
It would be a bit better if you had those examples each in their own methods.
They are unrelated blocks of code, which can exist by themselves, so might as well be placed in different places. Your 3 lists will be needed to be moved to private fields.

Reply:
sorry, this confuses me a little. First you said that they schould be in their own methods and later on you say they need to be private fields.

Reply:
Private fields fro the shared input- the 3 lists. It would be annoying to pass those 3 9 times.

Reply:
hmm, can I use a private field and still have the numbers as contents?

Reply:
solved